axon-server/administration/clustering.md

```diff
@@ -0,0 +1,51 @@
+# Clustering
+
+> Note - This feature is only available in Axon Server Enterprise
+
+## Introduction
+
+Axon Server Enterprise can be deployed as a _**cluster**_ to guarantee high availability. A cluster of Axon Server nodes will provide multiple connection points for \(Axon Framework-based\) client applications, and thus share the load of managing message delivery and event storage. Client applications will dynamically connect to a node in the cluster and automatically reconnect to another, should the node that they are currently connected to become unreachable.‌
```

Comment:
So, currently stating that "Axon Framework-based" applications can connect is fair, as we only provide a connector tied to Axon Framework. But, we are working on a non-Axon Framework specific version, which will also serve the purpose of a sample for other language implementations. 

Thus it might be nice to have a notice on this somewhere, that it's not purely Axon Framework apps which can connect to Axon Server. Not sure what the right place for this is though.

Reply:
We can include a reference to it once it is available. I will mention the non-Axon FRamework specific version in another part of the reference guide